package.json

```diff
@@ -19,8 +19,13 @@
     "preset": "jest-expo"
   },
   "dependencies": {
+    "@expo/videoplayer": "^0.4.0",
     "expo": "^27.0.1",
     "react": "16.3.1",
-    "react-native": "~0.55.2"
+    "react-hot-loader": "^4.3.6",
+    "react-native": "~0.55.2",
+    "react-native-video": "^3.2.0",
+    "react-native-youtube": "^1.1.0",
+    "react-player": "^1.6.4"
```

Comment:
package.json: Installation of dependencies enables the video to be embedded into the landing page.